src/NUnitFramework/framework/Api/FrameworkPackageSettings.cs

```diff
@@ -106,10 +106,15 @@ public static class FrameworkPackageSettings
         public const string DefaultTestNamePattern = "DefaultTestNamePattern";
 
         /// <summary>
-        /// Parameters to be passed on to the test
+        /// Parameters to be passed on to the tests, serialized to a single string which needs parsing. Obsoleted by <see cref="TestParametersDictionary"/>; kept for backward compatibility.
         /// </summary>
         public const string TestParameters = "TestParameters";
 
+        /// <summary>
+        /// Parameters to be passed on to the tests, already parsed into an IDictionary&lt;string, string>. Replaces <see cref="TestParameters"/>.
```

Comment:
Sorry - it's not! I presumed `&lt;string, string>` should be either all escaped or not escaped, but I just tried my 'correction' - and it doesn't compile, as I presume you already knew. Sorry, false alarm!